pkgs/development/misc/newlib/default.nix

```diff
@@ -1,10 +1,23 @@
-{ stdenv, fetchurl, buildPackages }:
+{ stdenv, fetchurl, fetchFromGitHub, buildPackages }:
```

Comment:
Likewise, I think an override would be nicer here.

Reply:
It could go in a separate file, taking "regular" newlib as an arg and overriding it, then put into place like here: https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L12495-L12513